tfs/src/main/java/hudson/plugins/tfs/rm/ReleaseWebHookName.java

```diff
@@ -0,0 +1,60 @@
+package hudson.plugins.tfs.rm;
+
+import com.cloudbees.plugins.credentials.common.StandardListBoxModel;
+import hudson.Extension;
+import hudson.model.AbstractDescribableImpl;
+import hudson.model.Descriptor;
+import hudson.util.ListBoxModel;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import java.util.logging.Logger;
+
+/**
+ *
+ * @author Kalyan
+ */
+public class ReleaseWebHookName extends AbstractDescribableImpl<ReleaseWebHookName> {
```

Comment:
if this is really reqd. suggest renaming as ReleaseWebHookReference?

Reply:
fixed.